src/ds/ds-motion-common.h

```diff
@@ -1,47 +1,15 @@
 // License: Apache 2.0. See LICENSE file in root directory.
-// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
 
 #pragma once
 
-#include "ds5-device.h"
+#include "ds-device-common.h"
+#include "core/video.h"
+#include "ds5/ds5-private.h"
+#include "ds6/ds6-private.h"
```

Comment:
In the old file we didn't have to include ds5-private.h
Now we do?

Reply:
This is needed because of the calib parsers that have been moved there (from ds5-motion.cpp)